ui-modules/blueprint-composer/app/components/catalog-saver/catalog-saver.modal.template.html

```diff
@@ -60,7 +60,7 @@ <h3 class="modal-title">{{title}}</h3>
                 <label class="control-label">Bundle ID</label>
                 <div class="input-group">
                     <span class="input-group-addon">catalog-bom-</span>
-                    <input ng-model="config.bundle" ng-disabled="state.saving" class="form-control" name="bundle" ng-pattern="state.pattern" autofocus placeholder="{{ defaultBundle || 'E.g. my-bundle-id' }}"/>
+                    <input ng-model="config.bundle" ng-disabled="isUpdate() || state.saving" class="form-control" name="bundle" ng-pattern="state.pattern" autofocus placeholder="{{ defaultBundle || 'E.g. my-bundle-id' }}"/>
```

Comment:
@ahgittin, instead of suppressing duplicate version check if ID is changed, I suggest to block bundle ID change all together when template update launched from the catalog

Changing bundle ID on template update creates a mess with versions in the catalog, where a "new" bundle holds information about previous versions that do not exist this *new* bundle, and vice versa.

I did not see how such a mess is possible to create via CLI.

Reply:
sounds reasonable